docs/src/man/doctests.md

```diff
@@ -221,6 +221,10 @@ julia> foo(2)
     and no state is shared between any code blocks.
     To preserve definitions see [Preserving Definitions Between Blocks](@ref).
 
+!!! note
+
+    If you are using the `@docs` or `@autodocs` macros to splice docstrings into your document, the `@meta` block must be in the markdown file that calls these macros and not within the docstrings themselves or they will be ignored.
```

Comment:
```suggestion
    If you rely on setup-code for doctests inside docstrings, included in the document with `@docs` or `@autodocs`, the `@meta` block must be in the markdown file that calls these macros and not within the docstrings themselves, otherwise they will be ignored.
```